feat(ins): add local storage (#4686)
scaleway/docs-content: pages/local-storage/index.mdx

---
meta:
  title: Local Storage Documentation
  description: Dive into Scaleway Local Storage with our quickstart guides, how-tos, tutorials and more.
---

<ProductHeader
   productName="Local Storage"
   productLogo="generic"
   description="The local volume of an Instance is an all-SSD-based storage solution, using a RAID array for redundancy and performance, hosted on the local hypervisor."
   url="/local-storage/quickstart"
   label="Local Storage Quickstart"
/>

## Getting Started

<Grid>
    <SummaryCard
        title="Quickstart"
        icon="rocket"

>         description="Learn how to create, connect to, and delete an Instance in a few steps."

Reviewer: This doesn't seem to be the right description

>         description="Learn how to create, manage and delte a Local Storage Volume a few steps."

Reviewer 2: ```suggestion
        description="Learn how to create, manage, and delete a Local Storage volume in a few steps."
```

        label="View Quickstart"
        url="/local-storage/quickstart/"
    />
    <SummaryCard
        title="Concepts"
        icon="info"

>         description="Core concepts that give you a better understanding of Scaleway local-storage."

Reviewer: ```suggestion
        description="Core concepts that give you a better understanding of Scaleway Local Storage."
```

        label="View Concepts"
        url="/local-storage/concepts/"
    />
    <SummaryCard
        title="How-tos"
        icon="help-circle-outline"

>         description="Check our guides to creating and managing local-storage and their features."

Reviewer: ```suggestion
        description="Check our guides to creating and managing Local Storage and its features."
```

        label="View How-tos"
        url="/local-storage/how-to/"
    />
</Grid>

<ClickableBanner
    productLogo="cli"
    title="Instance API"

>     description="Learn how to create and manage your Scaleway local-storage through the API."

Reviewer: ```suggestion
    description="Learn how to create and manage your Scaleway Local Storage through the API."
```

    url="https://www.scaleway.com/en/developers/api/instance/"

>     label="Go to local-storage API"

Reviewer: ```suggestion
    label="Go to Local Storage API"
```
